### ISSUE TITLE
"Septermber"?

### ISSUE THREAD

**Issue by fncll** [thisisparker/xword-dl#152]
This only seems to be a problem (so far) with McKinsey crosswords, but attempting to grab a September puzzle is failing due to a typo in the month:

```
(xword-dl) bash-3.2$ xword-dl --version
2023.12.2
(xword-dl) bash-3.2$ xword-dl mck -d 9/26/23
Unable to load https://www.mckinsey.com/featured-insights/the-mckinsey-crossword/septermber-26-2023
```
But the puzzle does exist at `https://www.mckinsey.com/featured-insights/the-mckinsey-crossword/september-26-2023`

**Comment by thisisparker** [comment]
LOL this is extremely my fault and fixable. Sorry about that! Will be fixed in the next release

**Comment by danielpunkass** [comment]
I feel there was a missed opportunity here to ask @thisisparker to "wake me up when Septermber ends..."

**Comment by thisisparker** [comment]
You ever look at a problem and think: "Why overcomplicate this? I'll go with the 'stupid' solution!" And then you smugly think about all the premature-optimizers on Stack Overflow who are winning code golf competitions but hey, you've _shipped_! And then it turns out you misspelled September?

**Comment by thisisparker** [comment]
Finally shipped this fix in `2024.7.20`
